src/main/java/com/ace/alfox/api/MapApi.java

```diff
@@ -0,0 +1,50 @@
+package com.ace.alfox.api;
+
+import com.ace.alfox.game.models.Player;
+import com.ace.alfox.lib.data.Database;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MapApi {
+
+  @Autowired private Database playerDatabase;
+
+  @GetMapping("/api/getMap")
```

Comment:
```suggestion
  @GetMapping("/api/map")
```
nit: probably don't need to repeat the verb used for fetching.

Reply:
Yes! Great point